### PR TITLE
updating to cowboy 2.5.0

### PR DESCRIPTION
**Pull request by alex-strizhakov**

### CODE REVIEW

**Comment by edgurgel** [comment]
Thanks for starting this! I'm really keen to get this merged! Really appreciated 🎉

File: .travis.yml [hunk 1]
```diff
@@ -3,10 +3,10 @@ notifications:
   recipients:
     - eduardo@gurgel.me
 elixir:
-  - 1.4.2
+  - 1.5.3
 otp_release:
   - 18.0
-  - 18.1
   - 19.0
+  - 20.0
```
Comment (rinpatch) [inline .travis.yml]: Maybe use a [build matrix](https://docs.travis-ci.com/user/build-matrix/) to test on newer OTP versions with supported elixir versions as well? like in https://github.com/edgurgel/httpoison/blob/master/.travis.yml

**Comment by alex-strizhakov** [comment]
@edgurgel  👋
Maybe you have side projects where we can test httparrot endpoints? 
I tested it with [tesla](https://github.com/teamon/tesla), but I assume that not all endpoints are used there.

**Comment by edgurgel** [comment]
Thanks! I will give it a try and fix if I find any issues! Cheers 🎉

**Comment by edgurgel** [comment]
I had to make a few changes but nothing major https://github.com/edgurgel/httparrot/commit/1f4f7161246702a52bba737335e012d57b433f8e Thanks! I will release a new version soon

**Comment by edgurgel** [comment]
Done! HTTPoison is already using HTTParrot for tests: https://github.com/edgurgel/httpoison/commit/ab7fce0a022d483833472ebaf91715eb4936c350

**Comment by alex-strizhakov** [comment]
Thanks for fast feedback! 😄

**Pull request merged**